packages/block-editor/src/components/colors-gradients/control.js

```diff
@@ -137,22 +127,46 @@ function ColorGradientControlInner( {
 						</legend>
 					) }
 					{ canChooseAColor && canChooseAGradient && (
-						<TabPanel
-							className="block-editor-color-gradient-control__tabs"
-							tabs={ TABS_SETTINGS }
-							initialTabName={
-								gradientValue
-									? TAB_GRADIENT.value
-									: !! canChooseAColor && TAB_COLOR.value
-							}
-						>
-							{ ( tab ) => renderPanelType( tab.value ) }
-						</TabPanel>
+						<div className="block-editor-color-gradient-control__tabs">
+							<Tabs
+								initialTabId={
+									gradientValue
+										? 'gradient'
+										: !! canChooseAColor && 'color'
+								}
+							>
+								<Tabs.TabList>
+									<Tabs.Tab id={ 'color' }>
```

Comment:
When passing a string prop, we should just pass it as a string, that way it won't be necessary to tell React to further evaluate it as an expression. That suggestion applies to the rest of the strings that are passed as expressions in this PR.

```suggestion
									<Tabs.Tab id="color">
```

Furthermore, is it possible that the ID we're passing here may already be in use on the same page because it's a rather common one? Should we prefix it with something?

Reply:
Good points, I should have added a comment to that bit.

Passing a string here will trigger ESLint's error asking for `useInstanceID`, which I don't actually want to do. I should probably add an ignore comment with an explanation rather than going around it in this admittedly opaque way.

`Tabs` applies `useInstanceId` internally, so there id clashes shouldn't happen but, this comment has me realizing I should really take another look at that part of the component's API, as it really isn't going to be clear to consumers that will run into this exact situation. Putting this on my todo/followup list for Tabs.

Reply:
Thanks for looking into it!